users/tests/test_api_registration.py

```diff
@@ -1,23 +1,17 @@
 import pytest
 from django.contrib.auth import get_user_model
 from django.test import Client
-from django.urls import reverse
 from rest_framework.exceptions import ErrorDetail
 
 User = get_user_model()
 
 
-@pytest.fixture()
-def register_url() -> str:
-    return reverse("api-user-registration")
-
-
 @pytest.mark.django_db()
 def test_registration_form_valid_data(
     client: Client,
     user_model: type[User],
     generated_password: str,
-    register_url: str,
+    api_register_url: str,
```

Comment:
Please revert. This is for registering via from, not API, as the name of the test states. 
We need new test for API registration.

Reply:
Or maybe the naming for the tests is completely wrong? If this is designed for API tests then `test_registration_form_valid_data` is at least misleading. In that case, please review all the tests in this file and fix. 
Also, all the tests in this file should be calling API endpoints, not "register_url" or similar.

Reply:
Ah.. on third look.. yes, test names are misleading,  please fix. It should be something like `test_registration_api_valid_data` - not `test_registration_form_valid_data`. You see the issue?